osm_validator/app.py

```diff
@@ -10,6 +15,11 @@
 
 async def build_application():
     app = web.Application()
+
+    fernet_key = Fernet.generate_key()
+    secret_key = base64.urlsafe_b64decode(fernet_key)
+    setup(app=app, storage=EncryptedCookieStorage(secret_key=secret_key))
```

Comment:
lets move to setting `SECRET_KEY = base64.urlsafe_b64decode(os.environ['SECRET_KEY'])`, to avoid session loses after restart